botocore/configprovider.py

```diff
@@ -127,7 +127,18 @@
         'AWS_S3_US_EAST_1_REGIONAL_ENDPOINT', None, None
     )
 }
-
+# A mapping for the proxy specific configuration vars. These are
+# used to configure how botocore interacts with proxy setups while
+# sending requests.
+DEFAULT_PROXIES_CONFIG_VARS = {
+    'proxy_ca_bundle': ('proxy_ca_bundle', 'AWS_PROXY_CA_BUNDLE', None, None),
+    'proxy_client_cert': (
+        'proxy_client_cert',
+        'AWS_PROXY_CLIENT_CERT', None, None),
+    'proxy_use_forwarding_for_https': (
+        'proxy_use_forwarding_for_https',
+        'AWS_PROXY_FORWARDING_FOR_HTTPS', None, None),
```

Comment:
Curious if we want to use `ensure_bool` for this one?

Reply:
I had that initially but I'm not sure we do. We start indiscriminately sending the kwarg to urllib3 as False. I defaulted us to sending nothing which seemed safer.